src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/RawlsDAO.scala

```diff
@@ -4,12 +4,23 @@ import com.typesafe.scalalogging.slf4j.LazyLogging
 import org.broadinstitute.dsde.firecloud.FireCloudConfig
 import org.broadinstitute.dsde.firecloud.model.AttributeUpdateOperations.AttributeUpdateOperation
 import org.broadinstitute.dsde.firecloud.model._
+import org.joda.time.DateTime
+import spray.http.OAuth2BearerToken
 
 import scala.concurrent.Future
 
 /**
   * Created by davidan on 9/23/16.
   */
+
+object RawlsDAO {
+  lazy val refreshTokenPath = FireCloudConfig.Rawls.authPrefix + "/user/refreshToken"
+  lazy val refreshTokenDatePath = FireCloudConfig.Rawls.authPrefix + "/user/refreshTokenDate"
+
+  def pathToUrl(path: String) = FireCloudConfig.Rawls.baseUrl + path
+  def groupPath(group: String): String = FireCloudConfig.Rawls.authPrefix + s"/user/group/$group"
```

Comment:
I am not a big fan of this pattern. It leads to code like ```RawlsDAO.pathToUrl(RawlsDAO.groupPath("group"))``` which seems to complex to me. When do you ever want RawlsDAO.groupPath or any other path on its own? We should just have Url defs/vals e.g. ```def groupUrl(group: String) = FireCloudConfig.Rawls.baseUrl + FireCloudConfig.Rawls.authPrefix + s"/user/group/$group"```